documentation/release_6.3.htm

```diff
@@ -28,11 +28,15 @@ <h3>New functionality</h3>
   <li>
     Data from GE Discovery MI systems in RDF9 should now be readable. TOF information on these scanners has also been added.
     However, projection data is currently still always returned as non-TOF (but list-mode data is read as TOF).<br>
-    <a href=https://github.com/UCL/STIR/pull/1503>PR #1503</a>
+    <a href=https://github.com/UCL/STIR/pull/1519>PR #1503</a>
   </li>
   <li>
     <tt>stir_timings</tt> has now an extra option to parse a par-file for a projector-pair.
   </li>
+  <li>
+    Duration in sinogram interfile/exam_info obtained from lm_to_projdata has the correct value if we unlist all the events. This is not true for Root files
+    <a href=https://github.com/UCL/STIR/pull/1503>PR #1503</a>
```

Comment:
This has to be 1519. Also, please use ROOT, use `<tt>lm_to_projdata</tt>` and put a `<br>` after that line.